File-Formats/PDF-to-image/Converting-PDF-pages-into-Images.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Converting PDF pages into images| Syncfusion
+description: Learn about Converting PDF pages into images using Syncfusion PdfToImageConverter.
+platform: file-formats
+control: PDF to image
+documentation: UG
+---
+
+# Converting PDF pages into images
+
+PdfToImageConverter allows exporting pages of a PDF document into the stream using the Convert method. This option helps convert PDF pages into images.
```

Comment:
Instead of This option helps convert PDF pages into images. use we can convert single page or multiple pages into images.

Reply:
Yes. I have modified as mentioned.